models/time_driven_static_connection.h

```diff
@@ -0,0 +1,148 @@
+/*
+ *  time_driven_static_connection.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+#ifndef TIME_DRIVEN_STATIC_CONNECTION_H
+#define TIME_DRIVEN_STATIC_CONNECTION_H
+
+/* @BeginDocumentation
+   Name: time_driven_static_synapse - Synapse type for time-driven static
+   connections.
+
+   Description:
+
+   time_driven_static_synapse does not support any kind of plasticity. It simply
+   stores
+   the parameters target, weight, delay and receiver port for each connection.
```

Comment:
I suggest to extend this documentation here with the information that are provided in the first post of this PR. So basically explaining what this new connection is good for and when and how it could be adapted to simplify prototype implementations of new learning rules.

Reply:
good point, i've updated the docstring of the `TimeDrivenStaticSynapse`. please have a look whether that description is understandable.

Reply:
Thanks! Looks good to me.